mypy/server/update.py

```diff
@@ -1008,3 +1008,7 @@ def extract_type_maps(graph: Graph) -> Dict[str, Dict[Expression, Type]]:
     # This is used to export information used only by the testmerge harness.
     return {id: state.type_map() for id, state in graph.items()
             if state.tree}
+
+
+def is_verbose(manager: BuildManager) -> bool:
+    return manager.options.verbosity > 1 or DEBUG_FINE_GRAINED
```

Comment:
I think this should use `>=` instead of `>` for this to be an identical transformation if `DEBUG_FINE_GRAINED` is false.